### ISSUE TITLE
Create Web Interface

### ISSUE THREAD

**Issue by tfilip** [tfilip/Proiect-MPS#1]

**Comment by tfilip** [comment]
I don't think we have enought time to finish this bonus until deadline. Should we focus on ML ? @TibiSann

**Comment by TibiSann** [comment]
> I don't think we have enought time to finish this bonus until deadline. Should we focus on ML ? @TibiSann

Since there is no time to finish the bonus, we should focus on what we can do.So we will work on the ML part.

**Comment by tfilip** [comment]
Ok, we will live this issue for the future
